include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4102,6 +4102,46 @@
       "capabilities" : [ "Addresses", "VariablePointers", "VariablePointersStorageBuffer" ],
       "version" : "1.4"
     },
+    {
+      "opname" : "OpColorAttachmentReadEXT",
+      "class"  : "Image",
+      "opcode" : 4160,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Attachment'" },
+        { "kind" : "IdRef", "name" : "'Sample'", "quantifier" : "?" }
+      ],
+      "extensions" : [ "SPV_EXT_shader_tile_image" ],
```

Comment:
For all the instructions remove the extension enable. The desired chain is extension enables capability which enables instruction.

Reply:
Ack.
Just for my info - when existing ops (OpRayQueryInitializeKHR as an example) _do_ reference the extension name, that's just historical 'mistakes'?

Reply:
Yes, we have an issue to try and add this to CI, but haven't gotten around to it yet. This way is better for tooling because it reads both those fields as enables for the instruction. So the parser would accept just having the extension and there would need to be extra code for the capability requirement.

Reply:
OK. Thanks for the info.